app/javascript/pages/components/gallery/BlankCalendarItem.jsx

```diff
@@ -1,18 +1,20 @@
 import React from 'react';
+import PropTypes from 'prop-types';
 
-class BlankCalendarItem extends React.Component {
-  render() {
-    return (
-      <li className="calendar-grid__cell">
-        <a href={this.props.link} className="calendar-item">
-          <h2 className="calendar-item__month">{this.props.month} {this.props.year}</h2>
-          <div className="calendar-item__box--blank">
-            <h3 className="calendar-item__title--blank">Coming Soon</h3>
-          </div>
-        </a>
-      </li>
-    );
-  }
-}
+const BlankCalendarItem = (props) => {
+  const { month } = props;
+  return (
+    <li className="calendar-item__wrapper">
+      <h2 className="calendar-item__month">{month}</h2>
+      <div className="calendar-item__box--blank">
```

Comment:
Typically with BEM convention you include both classes. See https://en.bem.info/methodology/quick-start/#modifier

Reply:
Upon further reading it looks like the blank "modifiers" are doing more than what modifiers are expected to do (it's a completely different layout/design) so I'm going to refactor this a bit so that `calendar-item__wrapper--blank` contains BEM classes like `blank-calendar-item__month` and `blank-calendar-item__box`. They are two different React components, after all.